src/themes/DarkModeTheme.ts

```diff
@@ -0,0 +1,42 @@
+import { extendTheme, themeTools } from 'native-base';
```

Comment:
rename the file `src/themes/DarkModeTheme.ts` to `src/themes/index.ts`
this will simplify import later...